App/src/components/NewCalendar.js

```diff
@@ -96,14 +96,25 @@ function NewCalendar({
   forecast, // the forecast data
   users, // the list of users to show
   selectedUsers = [], // the list of users to highlight
-  selectedDate = null // the date to highlight those users
+  selectedDate = null, // the date to highlight those users
+  shiftsClicked
 }) {
   const dates = Array.from({ length: 7 }, (_, index) => {
     const date = new Date(startDate);
     date.setDate(startDate.getDate() + index);
     return date;
   });
 
+  console.log(shiftsClicked);
+
+  const buttonCss = {
```

Comment:
If it's not going to be used in more than one place it's better to place declare this object where it's being used, or maybe create a component for the button to apply those styles and only render it as `<Button {...props} />`

Reply:
Thanks yes, I updated this file.

Reply:
You actually don't need to background color validation if you are now using `selectedDate` and `selectedUsers` then it will automatically color the content, just be sure to pass an array of numbers for the `selectedUsers`.